std/experimental/logger/core.d

```diff
@@ -1,17 +1,9 @@
 ///
 module std.experimental.logger.core;
 
-import std.array;
-import std.stdio;
-import std.conv;
 import std.datetime;
-import std.string;
-import std.range;
+import std.range.primitives;
 import std.traits;
```

Comment:
The biggest benefit for me personally is that they document explicitly what is being imported. This makes them easier to convert to convert to local imports and to whatever DIP1005 brings to the table (language change or something along the lines of https://github.com/dlang/druntime/pull/1756).

> Plus, selective top level imports isn't the current style in the rest of Phobos.

Where we are at the moment is not interesting to talk about. What matters is where we want to be and how to get there.